packages/storefront-lwc/src/modules/commerce/productDetail/productDetail.html

```diff
@@ -20,28 +20,7 @@ <h3 class="float-left">{product.name}</h3>
         </div>
 
         <div class="row">
-            <div class="product-images col-12 col-sm-6">
-                <div class="carousel slide" data-interval="0">
-                    <div class="carousel-inner" role="listbox">
-                        <template for:each={images} for:item="image">
-                            <div class={image.cssClass} key={image.link}>
-                                <img src={image.link} class="d-block img-fluid" alt={image.atl}>
-                            </div>
-                        </template>
-                    </div>
-
-                    <a class="carousel-control-prev" href="javascript: void 0;" role="button" data-slide="prev"
-                        onclick={handleCarousel}>
-                        <span class="fa icon-prev" aria-hidden="true"></span>
-                        <span class="sr-only">Previous</span>
-                    </a>
-                    <a class="carousel-control-next" href="javascript: void 0;" role="button" data-slide="next"
-                        onclick={handleCarousel}>
-                        <span class="fa icon-next" aria-hidden="true"></span>
-                        <span class="sr-only">Next</span>
-                    </a>
-                </div>
-            </div>
+            <commerce-product-image-carousel class="col-12 col-sm-6" images={product.images}></commerce-product-image-carousel>
```

Comment:
I think the layout bootstrap classes (col-12 and col-sm-6) should exist on the outside, so that this component can be reused for other layouts in other contexts.